src/api/image/Dockerfile

```diff
@@ -7,7 +7,9 @@ WORKDIR /app
 
 COPY --chown=node:node . .
 
-RUN npm install --only=production --no-package-lock
+RUN npm i -g pnpm
```

Comment:
It would be interesting later on to create our own image that does:

```Dockerfile
FROM node:lts-alpine
RUN npm i -g pnpm
```

And then we can use that in all microservices.

TODO!

Reply:
This is it write #2803, right ?

Reply:
Oh, I just saw it, sorry

Reply:
> It would be interesting later on to create our own image that does:
> 
> ```dockerfile
> FROM node:lts-alpine
> RUN npm i -g pnpm
> ```
> 
> And then we can use that in all microservices.
> 
> TODO!

If I understand correctly, we want to create and image that do the above and uploaded to docker repo, so we can pull it down the image when we need it right?